share/completions/mkdir.fish

```diff
@@ -0,0 +1,9 @@
+complete -c mkdir -s m -l mode --description 'Set file mode (as in chmod)' -x
+complete -c mkdir -s p -l parents --description 'Make parent directories as needed'
+complete -c mkdir -s v -l verbose --description 'Print a message for each created directory'
+complete -c mkdir -s Z --description 'Set SELinux security context of each created directory to the default type'
```

Comment:
Seems like `modinfo` is behave in a completely different way on Solaris... :-/

If I understand well, I should have for `mkdir`:

```fish
if command -s sestatus
    complete -c mkdir -s Z --description 'Set SELinux security context of each created directory to the default type'
    complete -c mkdir -l context --description 'Like -Z' -f
end
```

But, `--version` is only present with GNU tools, so I should also have:
```fish
if mkdir --version >/dev/null ^/dev/null
    complete -c mkdir -l version --description 'Output version'
end
```